packages/flutter_tools/test/general.shard/base/command_help_test.dart

```diff
@@ -0,0 +1,175 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/command_help.dart';
+import 'package:flutter_tools/src/base/platform.dart' show Platform, platform;
```

Comment:
This library doesn't exist anymore, so you'll need to import package:platform directly for the Platform type and `globals.dart` for the top level platform getter:

```
import 'package:platform/platform.dart';
import 'package:flutter_tools/src/globals.dart' as globals;
```